packages/phone-number-privacy/combiner/src/common/database/database.ts

```diff
@@ -0,0 +1,14 @@
+import knex, { Knex } from 'knex'
+import { CombinerConfig, DEV_MODE } from '../../config'
+
+export function initDatabase(config: CombinerConfig): Knex {
+  return knex({
+    client: 'pg',
+    connection: config.db,
+    debug: DEV_MODE,
+  })
+}
+
+export function getTransaction(db: Knex) {
```

Comment:
(Note: wanna revisit this), IMO this wrapper feels unnecessary and obfuscates what's going on 🤔 (it's also used in 1 place other than tests, and doesn't feel like it would save any time as is even if Knex changed trx object structure or anything like that)